GruntFile.js

```diff
@@ -79,13 +78,14 @@ module.exports = function (grunt) {
         testname: 'atlas',
         maxRetries: 10,
         maxPollRetries: 10,
-        'public': 'public',
+        public: 'public',
```

Comment:
Isn't it best to wrap reserved words in quotes when they are used as properties? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar